packages/babel-plugin/src/css-prop/__tests__/object-literal.test.tsx

```diff
@@ -729,4 +729,20 @@ describe('css prop object literal', () => {
 
     expect(actual).toInclude('{color:red}');
   });
+
+  it('should retain keys for mapped react components', () => {
+    const actual = transform(`
+        import '@compiled/react';
+
+        ['foo', 'bar'].map((str) => (
+          <div key={str} css={{ backgroundColor: 'blue' }}>
+            {str}
+          </div>
+        ));
+      `);
+
+    console.log(actual);
+    expect(actual).toMatch(/<CC key={.+}>/);
```

Comment:
You could be more explicit and replace this with:

```javascript
expect(actual).toIncludeMultiple([
  '<CC key="foo">',
  '<CS key="foo">',
  '<CC key="bar">',
  '<CS key="foo">',
]);
```

Or if you want to be a bit more programmatic:

```javascript
expect(actual).toIncludeMultiple(['foo', 'bar'].flatMap((name) => [
  `<CC key="${name}">`,
  `<CS key="${name}">`,
]));
```

Though if you remove the key on `CS` (refer to another comment), then it can be simplified to just `map`.